### PR TITLE
fix typos in openocd.gdb comments

### PR DESCRIPTION
**Pull request by Jonahss**

### CODE REVIEW

**Comment by Jonahss** [comment]
Loving the Discovery book so far! Setup was painless.

File: src/openocd.gdb [hunk 1]
```diff
@@ -29,7 +29,7 @@ break DefaultHandler
 # Set a breakpiont at HardFault
 break HardFault
 
-# Continue running and unill we hit the main breakpoint
+# Continue running and until we hit the main breakpoint
```
Comment (adamgreig) [inline src/openocd.gdb:32]:
```suggestion
# Continue running until we hit the main breakpoint
```

**Review by adamgreig — COMMENTED** [review]
Thanks! Looks good but I think we can get rid of "and" here too?

**Comment by Jonahss** [comment]
@adamgreig included your suggestion. Squashed into one commit.

**Review by adamgreig — APPROVED** [review]
thanks!

bors merge

**Pull request merged**